index.html

```diff
@@ -3,9 +3,10 @@
 <head>
   <meta charset="utf-8">
   <title>Game Time</title>
+  <link rel="stylesheet" href="/lib/index.css" media="screen" charset="utf-8">
 </head>
 <body>
-
+  <canvas tabindex='1' id='game' width='1000px' height='600px'></canvas>
```

Comment:
I wonder if tabindex will cause us problems when testing. We might need to focus on the canvas for every test??